src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/SwapEndpointSpec.groovy

```diff
@@ -1142,6 +1142,76 @@ switches"() {
         ]
     }
 
+    def "System reverts both flows if fails during rule installation when swapping endpoints"() {
+        given: "Two flows with different src switches and same dst"
+        def swPair1
+        def swPair2 = topologyHelper.switchPairs.find { second ->
+            swPair1 = topologyHelper.switchPairs.find { first ->
+                first.src != second.src && first.dst == second.dst
+            }
+        }
+        assert swPair1
+        assert swPair2
```

Comment:
if `swPair` is not found then this test will be failed
probably would be better to use  `assumeTrue`